content/article/enhancing-midi-hardware-with-perl.md

```diff
@@ -0,0 +1,507 @@
+{
+    "title"       : "Enhancing your MIDI devices with Perl",
+    "authors"     : ["john-barrett"],
+    "date"        : "2025-01-30",
+    "tags"        : ["midi","music"],
+    "draft"       : true,
+    "thumbnail"   : "/images/enhancing-midi-hardware-with-perl/midicamel.png",
+    "description" : "Add functions and features to MIDI devices by routing them via Perl",
+    "categories"  : "data"
+}
+
+## Introduction
+
+These days, even modestly priced MIDI hardware comes stuffed with features.
+You should expect a budget device to provide at least some of clock, sequencer,
```

Comment:
`s/at least some of clock/at least some kind of clock/`

Reply:
It's some of those features, not variants of those features. I'll have a think about how to better phrase it.